interface/python_binding/README.md

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2023, Shukant Pal and Contributors \
+SPDX-License-Identifier: Apache-2.0
+
+Made with love by [Shukant Pal](https://www.shukantpal.com/about) on his way to learning Python.
+
+# pyktx
+
+This Python package provides a Pythonic interface to libktx. It uses CFFI to generate the C bindings.
+
+## Building
+
+To build and test pyktx,
+
+```bash
+# Set LIBKTX_INSTALL_DIR if you've installed libktx at the default system location.
+# Otherwise set LIBKTX_INCLUDE_DIR, LIBKTX_LIB_DIR to wherever you've built libktx.
+cd ${PROJECT_DIR}/interface/python_binding
+KTX_RUN_TESTS=ON python3 buildscript.py
```

Comment:
What about just building? Presumably `python3 buildscript.py`. Are LIBKTX env vars necessary for building or just testing? Please clarify.

Reply:
1. Yes excluding the run tests variable won't run tests.

2. KTX env vars are necessary for building, running , and testing UNLESS you've already installed libktx in its typical system location (eg /usr/local or Program Files on windows)

Reply:
So basically if you the `cmake install` command on your machine, you can simply run

```bash
KTX_RUN_TESTS=ON python3 ./buildscript.py
```

otherwise you'd run

```bash
            LIBKTX_INCLUDE_DIR=${CMAKE_SOURCE_DIR}/include
            LIBKTX_LIB_DIR=${KTX_BUILD_DIR}/Debug
            KTX_RUN_TESTS=ON
            DYLD_LIBRARY_PATH=${KTX_BUILD_DIR}/Debug:$DYLD_LIBRARY_PATH
            LD_LIBRARY_PATH=${KTX_BUILD_DIR}/Debug:$LD_LIBRARY_PATH
        python3 buildscript.py
```

Reply:
Presumably you can use Release or Debug. Correct?

Reply:
That's right.